app/packs/src/apps/admin/devices/DeviceNovncTab.js

```diff
@@ -45,50 +44,51 @@ const DeviceNovncTab = () => {
   }
 
   return (
-    <Form className="form-with-columns">
-      <FormGroup validationState={device.valid_novnc_target} className="col-half">
-        <ControlLabel>Target *</ControlLabel>
-        <FormControl
+    <Form className="d-flex justify-content-between flex-wrap">
+      <Form.Group className="w-50 mb-3 pe-4">
+        <Form.Label className="fw-bold">Target *</Form.Label>
+        <Form.Control
           type="text"
           value={device.novnc_target ? device.novnc_target : ''}
+          className={device.valid_novnc_target}
           onChange={(event) => onChange('novnc_target', event.target.value)}
           placeholder="e.g. ws://localhost:8092/websockify"
         />
-      </FormGroup>
+      </Form.Group>
 
-      <FormGroup className="col-half">
-        <ControlLabel>Websockify Token</ControlLabel>
-        <FormControl
+      <Form.Group className="w-50 mb-3">
+        <Form.Label className="fw-bold">Websockify Token</Form.Label>
+        <Form.Control
           type="text"
           value={device.novnc_token ? device.novnc_token : ''}
           onChange={(event) => onChange('novnc_token', event.target.value)}
           placeholder="e.g. 000001"
         />
-      </FormGroup>
+      </Form.Group>
 
-      <FormGroup className="col-full">
-        <span className="fa fa-info-circle" aria-hidden="true">&nbsp;
-          <b>Current Target</b>&nbsp;
+      <Form.Group className="w-100 mb-3">
+        <span className="fa fa-info-circle" aria-hidden="true">
```

Comment:
There are a couple of things weird here.
The Form.Group Element contains nothing but text.
The aria-hidden attribute of the span hides the text from screen readers.
The h4 apparently is a headline for stuff that exist in the next Form.Group.
RenderCurrentTarget looks like a component in pointy brackets, we use curly ones for functions.
```
<p>
  <i className="fa fa-info-circle me-1" aria-hidden="true">
  <span className="fw-bold">Edited Target</span>
  {RenderCurrentTarget()}
  </p>
  ...
  <hr/>
  <Form.Group>
  <h4>RFB Credentials</h4>
  ...
  ```